src/workloads/execution/TimeSeriesLastpoint.yml

```diff
@@ -46,75 +61,191 @@ Actors:
   - *Nop
   - *Nop
   - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
 
-- Name: CreateIndex
+# Lastpoint query with ascending index on meta subfield and descending index on time.
+- Name: RunLastpointQueryWithMetaSubfieldAscendingTimeDescending
   Type: RunCommand
-  Threads: 1
+  Database: *db
   Phases:
   - *Nop
   - *Nop
-  - Repeat: 1
-    Database: *db
-    Operations:
-    - OperationName: RunCommand
-      OperationCommand:
-        createIndexes: Collection0
-        indexes:
-        - {key: {"metadata.sensorId": 1, timestamp: -1}, name: "MetaSubfieldAscendingTimeDescending"}
+  - LoadConfig:
```

Comment:
Could you add a phase here (and for the later queries as well) to test for when this query doesn't have an accompanying backing index? Note that after this the comments should be changed to reflect this.

For some context, according to Michael G the use cases we should care about for this project are (in order of importance):
1. when the collection has {meta: 1, time: 1} index
2. when the collection doesn't have any indices at all
3. all other index formulations - he doesn't think these will be very common, since MongoDB officially recommends config 1

Reply:
Good question - I think what you've done here is sufficient for this ticket scope but might be good to talk to Mike and see if there are any other indexes we should test against for the project.